examples/lb3-application/src/server.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/example-lb3-application
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ApplicationConfig} from '@loopback/core';
+import * as express from 'express';
+import {Request, Response} from 'express';
+import * as http from 'http';
+import pEvent from 'p-event';
+import * as path from 'path';
+import {CoffeeShopApplication} from './application';
+
+const loopback = require('loopback');
+const compression = require('compression');
+const cors = require('cors');
+const helmet = require('helmet');
+
+export class ExpressServer {
+  private app: express.Application;
+  public readonly lbApp: CoffeeShopApplication;
+  private server?: http.Server;
+
+  constructor(options: ApplicationConfig = {}) {
+    this.app = express();
+    this.lbApp = new CoffeeShopApplication(options);
+
+    // Middleware for LB3
+    this.app.use(loopback.favicon());
+    this.app.use(compression());
+    this.app.use(cors());
+    this.app.use(helmet());
+
+    // Expose the front-end assets via Express, not as LB4 route
```

Comment:
I find this comment misleading, we are not exposing front-end assets here, we are exposing LB4 REST API.